packages/core/docs/contributing/server-side-rendering.md

```diff
@@ -1,30 +1,30 @@
 # Server-side rendering
 
-In Vue Storefront we rely on Nuxt.js as a server-side rendering provider and we highly recommend this as well.
-However Nuxt.js has everything we need when it comes to SSR, we had to provide a bit more to this implementation.
+In Vue Storefront, we rely on Nuxt.js as a server-side rendering provider, and we highly recommend this as well.
+Although Nuxt.js has everything we need regarding SSR, we had to provide a bit more to this implementation.
 
-First of all. As we still wait for fully integration Vue 3 and Nuxt.js and the current SSR based on
+As we still wait for full integration of Vue 3 and Nuxt.js, and the current SSR based on
 composition-api didn't cover our needs, we had to temporarily sort it out on our own.
```

Comment:
Would you mind deleting this sentence? I'm sure we will not remember to do it once we upgrade to Vue 3 / Nuxt 3.